src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -0,0 +1,1091 @@
+							if (file) {
+								return joinPath(folderUri, file);
+							}
+						} catch (err) {
+							onUnexpectedError(err);
+						}
+						return undefined;
+					}));
+					arrays.coalesceInPlace(readmes);
+					if (!this.editorService.activeEditor) {
+						if (readmes.length) {
+							const isMarkDown = (readme: URI) => strings.endsWith(readme.path.toLowerCase(), '.md');
+							await Promise.all([
+								this.commandService.executeCommand('markdown.showPreview', null, readmes.filter(isMarkDown), { locked: true }),
+								this.editorService.openEditors(readmes.filter(readme => !isMarkDown(readme))
+									.map(readme => ({ resource: readme }))),
+							]);
+						} else {
+							await this.instantiationService.createInstance(WelcomePage).openEditor();
+						}
+					}
+				} else {
+					await this.instantiationService.createInstance(WelcomePage).openEditor();
+				}
+			}
+		}
+	}
+}
+
+
+function isWelcomePageEnabled(configurationService: IConfigurationService, contextService: IWorkspaceContextService) {
+	const startupEditor = configurationService.inspect(configurationKey);
+	if (!startupEditor.userValue && !startupEditor.workspaceValue) {
+		const welcomeEnabled = configurationService.inspect(oldConfigurationKey);
+		if (welcomeEnabled.value !== undefined && welcomeEnabled.value !== null) {
+			return welcomeEnabled.value;
+		}
+	}
+	return startupEditor.value === 'welcomePage' || startupEditor.value === 'readme' || startupEditor.value === 'welcomePageInEmptyWorkbench' && contextService.getWorkbenchState() === WorkbenchState.EMPTY;
+}
+
+export class WelcomePageAction extends Action {
+
+	public static readonly ID = 'workbench.action.showWelcomePage';
+	public static readonly LABEL = localize('welcomePage', "Welcome");
+
+	constructor(
+		id: string,
+		label: string,
+		@IInstantiationService private readonly instantiationService: IInstantiationService
+	) {
+		super(id, label);
+	}
+
+	public run(): Promise<void> {
+		return this.instantiationService.createInstance(WelcomePage)
+			.openEditor()
+			.then(() => undefined);
+	}
+}
+
+interface ExtensionSuggestion {
+	name: string;
+	title?: string;
+	description?: string;
+	id: string;
+	isKeymap?: boolean;
+	isCommand?: boolean;
+	isExtensionPack?: boolean;
+	icon?: string;
+	link?: string;
+	extensionPackExtensions?: any[];
+}
+
+
+const extensionPacks: ExtensionSuggestion[] = [
+	{
+		name: localize('welcomePage.adminPack', "SQL Admin Pack"),
+		title: localize('welcomePage.showAdminPack', "SQL Admin Pack"),
+		description: localize('welcomePage.adminPackDescription', "Admin Pack for SQL Server is a collection of popular database administration extensions to help you manage SQL Server"),
+		id: 'microsoft.admin-pack',
+		extensionPackExtensions: [
+			{
+				name: 'SQL Server Agent',
+				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-agent-extension?view=sql-server-2017',
+				icon: '../../../../../extensions/admin-pack/images/extension.png'
```

Comment:
Paths should be using path.join to create paths like this. 

But I'm also pretty sure this isn't going to work on a packaged build - these paths won't exist there. For the short-term fix I'd suggest just making a copy of that icon and sticking it with the rest of the icons here but we should also really be getting the image from the extension package itself in the future so that we don't have to update this any time an extension is updated with a new icon or we add a new extension or whatever.